### ISSUE TITLE
Mapping GPIO to key presses

### ISSUE THREAD

**Issue by scuba-hacker** [atomic14/esp32-zxspectrum#1]
Hi Chris. I don't
know if the code currently support this, but would it be possible to support using external switches connected via gpio so that custom controllers could be made? These could be mapped to kempston joystick inputs or even key presses.

**Comment by cgreening** [comment]
Yes - pretty straightforward. There are two QWIIC connectors and there are
quite a few I2C GPIO expander boards that would work nicely for this kind
of thing.
